lcUI/lcadviewer.cpp

```diff
@@ -16,33 +21,145 @@ LCADViewer::LCADViewer(QWidget *parent) :
     _scaleLineWidth(false),
     _backgroundPainter(nullptr),
     _documentPainter(nullptr),
-    _foregroundPainter(nullptr) {
+    _foregroundPainter(nullptr)
+     {
+
 
+   
+    QOpenGLContext *CC= QOpenGLContext::currentContext();
+    
     setMouseTracking(true);
     this->_altKeyActive = false;
     this->_ctrlKeyActive = false;
-	setCursor(Qt::BlankCursor);
+    setCursor(Qt::BlankCursor);
+
+   QSurfaceFormat format;
+    format.setMajorVersion(4);
+    format.setMinorVersion(6);
```

Comment:
Is this working on older OpenGL versions? Like 3.x?

Reply:
I have'nt check with any 3.x , my system has 4.4